### PR TITLE
Reverted idle.cpp so it functions again.

### PR DESCRIPTION
**Pull request by LeSqueed**
Prevent crash when audio device disconnects.

Should fix https://github.com/ErikReider/SwayAudioIdleInhibit/issues/10
and hopefully
https://github.com/ErikReider/SwayAudioIdleInhibit/issues/7

(not quite sure how to test that last one, but no more crashes when unplugging speaker usb cable on my  own laptop so...)

### CODE REVIEW

**Comment by ErikReider** [comment]
Still some segfaulting.

Running sway-audio-idle-inhibit and the code snippet below inside of a nested sway session results in a segfault if you pause/resume a audio source.
`swaymsg output $monitor unplug; swaymsg create_output $monitor`

File: src/pulse.cpp [hunk 1]
```diff
@@ -29,7 +29,7 @@ int Pulse::init(SubscriptionType subscriptionType,
 void Pulse::sink_input_info_callback(pa_context *, const pa_sink_input_info *i,
                                      int, void *userdata) {
   Data *data = (Data *)userdata;
-  if (i && !i->corked) data->activeSink = true;
+if (i && !i->corked && i->client != 0) data->activeSink = true;
```
Comment (ErikReider) [inline src/pulse.cpp:32]: Fix indentation and replace the 0 with `PA_INVALID_INDEX`

Comment (ErikReider) [reply src/pulse.cpp:32]: Should also be checked in the `source_output_info_callback` function too :)

Comment (LeSqueed) [reply src/pulse.cpp]: Thank you for giving me a way to test the issue with it crashing. Experimented a bit but not found a way yet that prevents the crash and still functions in inhibiting.

Might tinker some more if I find the time in between my regular work schedule. Certainly an interesting problem to look into further.

Comment (ErikReider) [reply src/pulse.cpp]: I'll merge this in the mean time :)

**Pull request merged**